hack/make-rules/test-integration.sh

```diff
@@ -41,27 +41,33 @@ KUBE_TEST_ARGS=${KUBE_TEST_ARGS:-}
 kube::test::find_integration_test_dirs() {
   (
```

Comment:
Looks like my top-level comment got lost, sorry -- the changes look fine, some weirdness in the stuff around them from pre-existing code, not sure if you want to touch it or not.